mock_tests/test_collection.py

```diff
@@ -370,3 +374,39 @@ def test_node_with_timeout(
 
     nodes = client.cluster.nodes(output=output)
     assert nodes[0].status == "TIMEOUT"
+
+
+def test_backup_cancel_while_create(
+    weaviate_no_auth_mock: HTTPServer, start_grpc_server: grpc.Server
+) -> None:
+    client = weaviate.connect_to_local(
+        port=MOCK_PORT,
+        host=MOCK_IP,
+        grpc_port=MOCK_PORT_GRPC,
+    )
+
+    backup_id = "id"
+
+    weaviate_no_auth_mock.expect_request("/v1/backups/filesystem").respond_with_json(
+        {
+            "collections": ["backupTest"],
+            "status": "CANCELED",
+            "path": "path",
+            "id": backup_id,
+        }
+    )
```

Comment:
Can we make the `status` returned here be `STARTED` so that it is clear that the backup creation succeeds with only subsequent calls to the status endpoint, as below, returning `CANCELED`? I think it would help make the test clearer that it's the `get_create_status` call that returns `CANCELED` and triggers the exception raising